t/app/controller/waste.t

```diff
@@ -1591,6 +1591,48 @@ FixMyStreet::override_config {
 
     };
 
+    subtest 'staff create new subscription - payment failed' => sub {
+        $mech->clear_emails_ok;
+        $mech->get_ok('/waste/12345/garden');
+        $mech->submit_form_ok({ form_number => 2 });
+        $mech->submit_form_ok({ with_fields => { existing => 'no' } });
+        $mech->content_like(qr#Total to pay now: £<span[^>]*>0.00#, "initial cost set to zero");
+        $mech->content_lacks('password', 'no password field');
+        $mech->submit_form_ok({ with_fields => {
+                current_bins => 0,
+                new_bins => 1,
+                name => 'Test McTest',
+                email => 'test@example.net'
+        } });
+        $mech->content_contains('Test McTest');
+        $mech->content_contains('£20.00');
+        $mech->content_contains('1 bin');
+        # external redirects make Test::WWW::Mechanize unhappy so clone
+        # the mech for the redirect
```

Comment:
Can this comment (and on L1696) be removed now as the clone doesn't seem to happen?